client/src/App.js

```diff
@@ -36,3 +38,23 @@ class App extends Component {
 }
 
 export default App;
+
+// const mapStateToProps = state => ({
```

Comment:
Let's try not to commit commented out code! Keep our files clean 💪

Reply:
Makes sense! I'll fix that.